Machine-Learning/README.md

```diff
@@ -0,0 +1,106 @@
+# Machine Learning
+
+## Introduction  
+
+- [What is Machine Learning? -IBM.com](https://www.ibm.com/topics/machine-learning)
+- [What is Machine Learning? -MIT](https://mitsloan.mit.edu/ideas-made-to-matter/machine-learning-explained)
+
+## Getting Started with Python
+
+- [The Python tutorial -python.org](https://docs.python.org/3/tutorial/)
+- [Python in One Short -FreeCodeCamp(Youtube)](https://youtu.be/eWRfhZUzrAch)
+- [Python for absolute Begineer  -Corey Schafer(Full playlist including frameworks  and libraries)](https://youtube.com/playlist?list=PL-osiE80TeTt2d9bfVyTiXJA-UTHn6WwU)
+- [Intermidiate python in 6 hours -FreeCodeCamp](https://youtu.be/HGOBQPFzWKo)
+- [Python Tutorial -Tutorials point](https://www.tutorialspoint.com/python/index.html)
```

Comment:
This resource doesn't exist.
Link leads to a 404 page.

Reply:
Fixed the link issue actually, it was [https://www.tutorialspoint.com/python/index.htm](url)

Reply:
Fixed the link issue actually it was [https://www.tutorialspoint.com/python/index.htm](url)

Reply:
Fixed the link issue